server/sonar-server/src/test/java/org/sonar/server/organization/DefaultOrganizationProviderRule.java

```diff
@@ -19,52 +19,24 @@
  */
 package org.sonar.server.organization;
 
-import com.google.common.base.Preconditions;
 import org.junit.rules.ExternalResource;
-import org.sonar.core.util.UuidFactoryImpl;
+import org.sonar.db.DbTester;
 
 public class DefaultOrganizationProviderRule extends ExternalResource implements DefaultOrganizationProvider {
```

Comment:
I expect there will be need for an implementation which isn't based on `DbTester` but you're right, it will be created when needed. Do not rename the class.